### PR TITLE
create contact page

### PR DESCRIPTION
**Pull request by liubamiron**
create contact page

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (egorbwork) [inline composer.json:1]: interesting why file was not present in project by default.

File: contacts/contactUs.blade.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            <form action="{{ route('contactUs.send') }}" method="POST" name="contact-form">
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="email">Email</label>
+                        <input type="email" value="{{old('email')}}" required="required" class="form-control" name="email" id="email" placeholder="Email">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <label for="name">Nume</label>
+                        <input type="text" value="{{old('name')}}" required="required" class="form-control" name="name" id="name" placeholder="Nume">
+                        @error('name')
```
Comment (egorbwork) [inline contacts/contactUs.blade.php]: please discard, it is already a block for all error messages.

File: controllers/ContactUsController.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{
+    public function view(){
+        return view('contacts.contactUs');
+    }
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        // dd($request->validated());
+
+        $callback = function (string $test): string {
```
Comment (egorbwork) [inline controllers/ContactUsController.php]: please be attentive on Monday lesson it was removed in the end.

File: controllers/ContactUsController.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                'email' => $data['email'],
+                'name' => $data['name'],
+                'districts' => $data['districts'],
+                'department' => $data['department'],
+                'messageText' => $data['message'],
+           
+            ],
+            function (Message $message) use ($data) {
+                $message->subject('message from ' . $data['email']);
+                $message->to('tech@baloon.app');
```
Comment (egorbwork) [inline controllers/ContactUsController.php]: please adjust to and from values to your project baloon word doesn't look good.

File: http/request/ContactUsRequest.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<?php
+ 
+namespace  App\Http\Request;
```
Comment (egorbwork) [inline http/request/ContactUsRequest.php:3]: Folder name should be called `Request` and not `request` it matters.

**Review by egorbwork — CHANGES_REQUESTED** [review]
Looks not bad, but some fixes are required.

File: http/request/ContactUsRequest.php (hunk 5 above)
Comment (liubamiron) [reply http/request/ContactUsRequest.php:3]: I changed request into Request

File: composer.json (hunk 1 above)
Comment (liubamiron) [reply composer.json:1]: put all project files in git ?

File: contacts/contactUs.blade.php (hunk 2 above)
Comment (liubamiron) [reply contacts/contactUs.blade.php]: removed error https://github.com/liubamiron/bootcamp-project/pull/9/commits/5b9b85772f7cfb3132b41b2fb39345f804a02bdd

File: controllers/ContactUsController.php (hunk 3 above)
Comment (liubamiron) [reply controllers/ContactUsController.php]: commented $callback  https://github.com/liubamiron/bootcamp-project/pull/9/commits/f91173f1c42186badb07ca9da1c9b0879198c481

File: controllers/ContactUsController.php (hunk 4 above)
Comment (liubamiron) [reply controllers/ContactUsController.php]: change email https://github.com/liubamiron/bootcamp-project/pull/9/commits/2cd09b59b361372a4c592e4d0b7b781f0bafd8a1

File: http/request/ContactUsRequest.php (hunk 5 above)
Comment (liubamiron) [reply http/request/ContactUsRequest.php:3]: changed folder name in project, but I can't change it in git. /home/miron/bootcamp-project/app/Http/Request/ContactUsRequest.php

File: controllers/ContactUsController.php [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+{
+    public function view(){
+        return view('contacts.contactUs');
+    }
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        // dd($request->validated());
+
+        // $callback = function (string $test): string {
```
Comment (egorbwork) [inline controllers/ContactUsController.php:21]: preferable to remove this code.

File: http/request/ContactUsRequest.php (hunk 5 above)
Comment (egorbwork) [reply http/request/ContactUsRequest.php:3]: if you use linux than you should not have this problem. In other case, I recommend to move file remove folder and create it again copying this file.

**Review by egorbwork — CHANGES_REQUESTED** [review]
Let's finish all requirements till the end.

File: controllers/ContactUsController.php (hunk 6 above)
Comment (liubamiron) [reply controllers/ContactUsController.php:21]: deleted, in the next part, you can see it https://github.com/liubamiron/bootcamp-project/pull/10/commits/c53453f1ee5fefd6fe7897d81b04a9322f8b41e8.

File: controllers/ContactUsController.php (hunk 3 above)
Comment (egorbwork) [reply controllers/ContactUsController.php]: Should be removed and not commented. Will ignore as it is removed in the next pr.

**Review by egorbwork — APPROVED** [review]
Not really workable for future project(folder names problem). But will approve to not create blocker.

**Pull request merged**